kubetest/kops.go

```diff
@@ -137,6 +137,9 @@ func (k kops) Up() error {
 	if k.image != "" {
 		createArgs = append(createArgs, "--image", k.image)
 	}
+	if os.Getenv("KOPS_ARGS") != "" {
```

Comment:
Please make this a kubetest flag. `--kopsArgs` `--kops-args` or something similar.